src/openloco/industry.cpp

```diff
@@ -60,7 +60,7 @@ namespace openloco
                             if (bl == 0 || bl != obj->var_EA)
                             {
                                 var_DB++;
-                                if ((!(obj->var_E4 & 0x10000000) && (surface->data()[4] & 0xE0) == 0) || find_5(surface))
+                                if ((!(obj->flags & 0x10000000) && (surface->data()[4] & 0xE0) == 0) || find_5(surface))
```

Comment:
Add a constant for this flag 0x10000000. Are we sure about this value? [locotool](https://github.com/jawsper/locotool/blob/master/libloco/objects.py#L800) only goes up to `800000`